src/main/scala/codacy/plugins/test/ITest.scala

```diff
@@ -106,7 +106,7 @@ trait ITest {
 
     if (otherPatternsResults.nonEmpty) {
       Printer.red(s"Some results returned were not requested by the test and were discarded!")
-      Printer.white(s"""
+      println(s"""
```

Comment:
shouldn't we use a Logging framework?

Reply:
Airframe is compatible with Graal:
https://github.com/wvlet/airframe/tree/master/airframe-log